### ISSUE TITLE
Confused by the loss function.

### ISSUE THREAD

**Issue by JinYang88** [theeluwin/pytorch-sgns#1]
In your code, you minimized `-(oloss + nloss).mean()`

which means (oloss+nloss) should be large.
So, "oloss become large and nloss become small " is expected.

Although -(oloss+nloss) decrease,  I got oloss become small and nloss become large, how so?

**Comment by theeluwin** [comment]
Thank you for the feedback.
Can you provide a reduced, reproducible case sample? Like, small dataset and a configuration for it.

**Comment by gongchenooo** [comment]
When computing `nloss`, the author uses function `.neg` to make the nloss smaller when training.
